tools/gcp_cli.py

```diff
@@ -275,6 +275,19 @@ def ListBucketObjects(args: 'argparse.Namespace') -> None:
         obj.get('contentType', 'Unknown Content-Type')))
 
 
+def GetBucketSize(args: 'argparse.Namespace') -> None:
+  """Get the size of a GCS bucket.
+
+  Args:
+    args (argparse.Namespace): Arguments from ArgumentParser.
+  """
+  gcs = gcp_storage.GoogleCloudStorage(args.project)
+  results = gcs.GetBucketSize(args.path)
```

Comment:
`results = gcs.GetBucketSize(bucket_name=args.path)`? See above for renaming the parameter.

Reply:
Ack, should be WAI now that it's a required param.